packages/suite/src/views/wallet/sign-verify/index.tsx

```diff
@@ -167,16 +167,18 @@ const SignVerify = () => {
     }, [isSignPage, formValues.message, formValues.address, formValues.signature]);
 
     const onSubmit = async (data: SignVerifyFields) => {
+        console.log(data);
+
         const { address, path, message, signature, hex, isElectrum } = data;
 
-        if (isSignPage) {
+        if (isSignPage && path !== undefined) {
```

Comment:
This is a hacky if that is required due to bad abstraction. Sign and Verfy forms shall be separate.

This is something that shall never happen (on Sign, the yup validation shall make `path` required`).